azure-pipelines.windows.yml

```diff
@@ -152,6 +155,6 @@ stages:
 
     - job: ort
       dependsOn: onnx
-      timeoutInMinutes: "1440"
+      timeoutInMinutes: "240"
```

Comment:
That may be too tight.
I would recommend to only use 1/2/6/12/24/48h, and make sure you have have plenty of head room if there's no pressure of early termination.

Reply:
The typical build time is around 2hrs for ort and 1hr for opencv on Standard_D4_v4 (4CPU, 16GB MEM). I'm reducing the timeout in case there's unexpecting hanging. Anyway, I'll increase both timeout to 6hrs. That's should be sufficient.

Reply:
Values here should be general, not specific to a machine setup.
Stage timeout is not a good way to detect hanging due to its bad sensitivity (positive/negative not separable).
Features like "timeout for silence in log" or "command-specific timeout" work better.

6h should be enough, assuming we're not using single-core machine and we have good network.
I'm OK with these assumptions for now.